docker/build_scripts/build.sh

```diff
@@ -51,42 +51,33 @@ esac
 # Decided not to clean at this point: https://github.com/pypa/manylinux/pull/129
 yum -y update
 
-# EPEL support (for cmake28 & yasm)
-yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
-
-case $AUDITWHEEL_ARCH in
-x86_64)
-  # Install devtoolset-8
-  yum -y install centos-release-scl
-  yum -y install \
-    devtoolset-8-binutils \
-    devtoolset-8-gcc \
-    devtoolset-8-gcc-c++ \
-    devtoolset-8-gcc-gfortran \
-  ;;
-i686)
-  # Install devtoolset-7 (binutils, gcc, gcc-c++, gcc-gfortran)
-  devtoolset7s=(
-    "devtoolset-7-runtime-7.1-4.el6.i686.rpm"
-    "devtoolset-7-binutils-2.28-11.el6.i686.rpm"
-    "devtoolset-7-gcc-7.3.1-5.10.el6.i686.rpm"
-    "devtoolset-7-libstdc++-devel-7.3.1-5.10.el6.i686.rpm"
-    "devtoolset-7-gcc-c++-7.3.1-5.10.el6.i686.rpm"
-    "devtoolset-7-libquadmath-devel-7.3.1-5.10.el6.i686.rpm"
-    "devtoolset-7-gcc-gfortran-7.3.1-5.10.el6.i686.rpm"
-  )
-  for rpm in "${devtoolset7s[@]}"; do
-	  yum install -y "https://www.repo.cloudlinux.com/cloudlinux/6.10/sclo/devtoolset-7/i386/$rpm"
-  done
-  ;;
-esac
+DEVTOOLSET8_TOOLCHAIN_DEPS="devtoolset-8-binutils devtoolset-8-gcc devtoolset-8-gcc-c++ devtoolset-8-gcc-gfortran"
+DEFAULT_TOOLCHAIN_DEPS="gcc gcc-c++ gcc-gfortran"
+if [ "${AUDITWHEEL_ARCH}" == "x86_64" ]; then
+    # Software collection (for devtoolset-8)
+    yum -y install centos-release-scl
+    # EPEL support (for yasm)
+    yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    YASM=yasm
+    TOOLCHAIN_DEPS=${DEVTOOLSET8_TOOLCHAIN_DEPS}
+elif [ "${AUDITWHEEL_ARCH}" == "aarch64" ]; then
+    # Software collection (for devtoolset-8)
+    # but needs erasing to be able to install...
+    # install->erase is required, removing the 2 command lines below fails.
+    yum -y install centos-release-scl
```

Comment:
```
yum -y install centos-release-scl-rh
```
As there is no centos/sclo/aarch64/sclo directories. This way no need for "yum erase"

Reply:
https://bugs.centos.org/view.php?id=16644 filled.